### PR TITLE
Feature refactor

### PR DESCRIPTION
**Pull request by lucasmello**

### CODE REVIEW

File: driloader/browser/exceptions.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    """ BrowserDetectionError """
+    def __init__(self, message, cause):
+        """Init method
+        Sets superclass arguments up.
+        Sets the cause of exception up.
+        """
+        super(BrowserDetectionError, self).__init__(message)
+        self.cause = cause
+
+    # def __str__(self):
```
Comment (jonathadv) [inline driloader/browser/exceptions.py]: Please remove useless commend

File: driloader/browser/exceptions.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    """ BrowserDetectionError """
+    def __init__(self, message, cause):
+        """Init method
+        Sets superclass arguments up.
+        Sets the cause of exception up.
+        """
+        super(BrowserNotSupportedError, self).__init__(message)
+        self.cause = cause
+
+    # def __str__(self):
```
Comment (jonathadv) [inline driloader/browser/exceptions.py]: Please remove useless comment

File: driloader/browser/firefox.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        except ImportError:
+            from winreg import OpenKey, QueryValue, HKEY_LOCAL_MACHINE, HKEY_CURRENT_USER
+
+        import shlex
+
+        keys = (r"SOFTWARE\Classes\FirefoxHTML\shell\open\command",
+                r"SOFTWARE\Classes\Applications\firefox.exe\shell\open\command")
+
+        for path in keys:
+            try:
```
Comment (jonathadv) [inline driloader/browser/firefox.py]: This code is pretty the same inside the both try-except.
How about move it to a different function and a call it recursively?

File: driloader/__main__.py [hunk 4]
```diff
@@ -37,18 +38,15 @@ def __init__(self, message, cause):
...
         self.cause = cause
 
-    def __str__(self):
-        return 'Error: {}.\nCause: {}'.format(self.args[0], self.cause)
+    # def __str__(self):
+    #     return 'Error: {}.\nCause: {}'.format(self.args[0], self.cause)
 
 
-class DriloaderCommands():
+class DriloaderCommands:
```
[the remark this answers is not in the document]
Comment (lucasmello) [reply driloader/__main__.py]: Sure, we can do that :)

**Pull request merged**